src/main/java/RegularTriangle.java

```diff
@@ -0,0 +1,55 @@
+import static java.lang.Math.*;
+import static java.lang.Math.sqrt;
+
+public class RegularTriangle implements Figure{
+
+    private final double sideLength;
+
+    public double getSideLength() {
+        return sideLength;
+    }
+
+
+    public double getheight () {
```

Comment:
zmiana nazwy na getHeight()?
```suggestion
    public double getHeight () {
```